index.html

```diff
@@ -10,15 +10,15 @@
         href="https://cdn.jsdelivr.net/gh/loadingio/loading.css@v2.0.0/dist/loading.min.css" />
     <link href="./style.css" rel="stylesheet" />
 </head>
-
 <body>
     <header>
         <div class="wrapper">
             <nav>
                 <ul>
                     <li class="calendare">
                         <a id="calendar" class=" calendar-icon active" href="#"><img
-                                src="./calendar-plus-solid.svg"></a>
+                                src="./calendar-plus-solid.svg"></a><a id="circle" class="circle-icon" href="#"><img
```

Comment:
I believe you didn't stage+commit+push the `circle-solid` image, because on my computer I cannot see it.

Nevertheless, why don't you try the following:

- Instead of an image for the dot, create an empty div and use CSS border + bacgrkound properties. By default, it has opacity 0. When the link is active, it sets opacity to 1.  
- The icon itself does not need to disappear (display: none). By doing that, you remove it from the layout and it breaks the animations. So, it's better to also use the opacity tricky.

Another element missing is the link text. To keep things easy, the opacity trick can also be used.

I hope this helps you!